scripts/export-framework-applicant-details.py

```diff
@@ -37,7 +37,7 @@
     client = DataAPIClient(get_api_endpoint_from_stage(STAGE), API_TOKEN)
     now = datetime.datetime.now()
 
-    filename = OUTPUT_DIR + os.sep if OUTPUT_DIR else ""
+    filename = (OUTPUT_DIR if OUTPUT_DIR else ".") + os.sep
```

Comment:
I've been using a slightly difference pattern for other scripts, which is that if OUTPUT_DIR is not specified, then write to stdout instead of writing a file at all.

I think this is preferable because probably you either want to be explicit about the target location ('.' could be anything), or you are probably debugging stuff in which case stdout is preferable anyway.

For an example see https://github.com/alphagov/digitalmarketplace-frameworks/pull/417/files#diff-9e8880f84bd814f2dc1eeef5d4a11361